.github/workflows/build.yml

```diff
@@ -19,12 +19,10 @@ jobs:
         with:
           go-version: ^1.18
 
-      - name: Set up Rust Nightly
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: nightly-2022-05-23
-          components: rustfmt, clippy
+      - name: Set up Rust
```

Comment:
The nightly is the one that needs to be used. Also, it should be a date pinned one, so that we can preserve the reproducible build promise we have.

See also https://github.com/parca-dev/parca-agent/pull/431/files

Sorry for the confusion here, we just merge the Rust pipeline and not everything is stabilized or documented yet.

Reply:
> The nightly is the one that needs to be used

Yes, the date-pinned nightly channel is set in the root `rust-toolchain.toml` file, the `rustup` commands will respect it. Sorry if it was not clear, it got lost in all the edits I did.

You can verify in the pipeline:

```console
info: syncing channel updates for 'nightly-2022-05-23-x86_64-unknown-linux-gnu'
info: latest update on 2022-05-23, rust version 1.63.0-nightly (b2eed72a6 2022-05-22)
```

> Sorry for the confusion here, we just merge the Rust pipeline and not everything is stabilized or documented yet.

No worries, I noticed, quite happy you made the jump from C to Rust!